randomerz/Slider
# Adds tune when final Caves puzzle is solved and fixes dialogue typo

## Description

Created a tune that plays after solving the final puzzle in Caves (specifically after Shake 1 when the Archeologist begins to reveal the scroll). Also fixes a typo that kept the word `scroll` from being colored orange in this scene.

## Related Task

Programming -> Polish -> SFX

## How Has This Been Tested?



- Navigate to Assets/Scenes/Dev/Avery/Caves
- Run the scene, and then in the debug tool run `ES 8`, wait for the tiles to spawn, then run `Switch` to make sure the building switch is turned on
- Talk to Archaeologist and then solve the eight puzzle (just move the border pieces if you used `ES 8`)
- Talk to the Archaeologist again. On the first screen shake, the background music should mute, and the tune will play. After the tune plays, the music should resume.
- When the Archaeologist says "A scroll?!", the word "scroll" should be in orange.

## Screenshots (if appropriate):
